### PR TITLE
Add Bash completion

### PR DESCRIPTION
**Pull request by FedericoCeratto**

### CODE REVIEW

**Comment by dom96** [comment]
Nice!

File: nimble.bash-completion [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# vim: filetype=sh
+
+_nimble()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    local prev=${COMP_WORDS[COMP_CWORD-1]}
+    COMPREPLY=()
+    _init_completion || return
```
Comment (oderwat) [inline nimble.bash-completion]: This line gives an error on my system (OSX). I simply removed it.. Is this working on Linux?

Comment (dom96) [inline nimble.bash-completion]: @FedericoCeratto ^

**Comment by FedericoCeratto** [comment]
@oderwat it does.

**Comment by oderwat** [comment]
Well Macs bash completion doesn't have this for the installed version of the completion. One could add a minimal version which looks like this:

```
__my_init_completion()
{
    COMPREPLY=()
    _get_comp_words_by_ref cur prev words cword
}
```

But afais it is not at all needed for your completion code and works without it. Why not remove it or just call it if it is available?

**Pull request merged**